client/src/components/ui/itemrow/InboxRow.svelte

```diff
@@ -7,23 +7,37 @@
     import { IconSize, ContextPayload, RowType, Events } from '../../types.ts';
     import { Document } from '../../../../../model/src/document.ts';
     import { Category } from '~model/category.ts';
+    import { Snapshot } from '~model/snapshot.ts';
 
     export let iconSize: IconSize;
     export let doc: Document['id'];
     export let category: Category['name'];
     export let title: Document['title'];
+    export let creation: Snapshot['creation'];
 
     const dispatch = createEventDispatcher();
     const rowEvent: ContextPayload = {
         ty: RowType.Inbox,
         id: doc,
     };
+
+    function redirectHandler() {
+        window.location.href = `/track?id=${doc}`;
+    }
```

Comment:
At this point, we should extract this as a helper function already.

```ts
function goToTrackingPage(doc: Document['id']) {
    location.href = '/track?id=' + doc;
}
```

Reply:
I am not sure if I can access the `window` API from an external Typescript `util.ts` file. This is the reason why kept the code in the `svelte` file.

Reply:
Yes, it is possible. That's exactly how the `src/api/` wrappers are implemented (i.e., `fetch === window.fetch`). 👌